Add suspended job indicator (#616)
sindresorhus/pure: readme.md

- `prompt:error` (red) - The `PURE_PROMPT_SYMBOL` when the previous command has *failed*.
- `prompt:success` (magenta) - The `PURE_PROMPT_SYMBOL` when the previous command has *succeeded*.
- `prompt:continuation` (242) - The color for showing the state of the parser in the continuation prompt (PS2). It's the pink part in [this screenshot](https://user-images.githubusercontent.com/147409/70068574-ebc74800-15f8-11ea-84c0-8b94a4b57ff4.png), it appears in the same spot as `virtualenv`. You could for example matching both colors so that Pure has a uniform look.

> - `suspended_jobs` (red) - The four pointed star showing jobs runs in the background.

Reviewer: ```suggestion
- `suspended_jobs` (red) - The `✦` symbol indicates that jobs are running in the background.
```
